sqflite/android/src/main/java/com/tekartik/sqflite/Database.java

```diff
@@ -391,25 +390,20 @@ private boolean executeOrError(Operation operation) {
         if (LogLevel.hasSqlLevel(logLevel)) {
             Log.d(TAG, getThreadLogPrefix() + command);
         }
-        boolean operationInTransaction = Boolean.TRUE.equals(operation.getInTransactionChange());
-
+        Boolean operationInTransaction = operation.getInTransactionChange();
```

Comment:
How about transactionDepth? I can change it back if it is not a good name :)

Reply:
I remember having such name before but SQLite does not support nested transaction so that's not really needed, but that's fine. I will test with sending multiple BEGIN to see the behavior